README.rst

```diff
@@ -205,6 +205,11 @@ Tests
    make bootstrap
    make test
 
+Testbed suite
+^^^^^^^^^^^^^
+
+A testbed suite designed to test API changes and experimental features is included under the `testbed` directory. For more information, see the `Testbed README <testbed/README.rst>`_.
+
```

Comment:
I have no idea why we're using `.rst` for readme... seems like the only OT repo that does that :-(

Reply:
It's very common in Python projects (or at least it was) using `.rst` instead of Markdown. Some examples with widely used libraries:
* https://github.com/pallets/flask/blob/master/README.rst
* https://github.com/django/django/blob/master/README.rst

I don't have strong opinions on that, since what is important to me is the content of the README and keeping it consistent across other OT repos is a good idea.